src/pages/home/ReportScreen.tsx

```diff
@@ -313,12 +313,13 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
     }, [report]);
 
     const onBackButtonPress = useCallback(() => {
-        if (isInNarrowPaneModal) {
+        const backTo = route?.params?.backTo as string;
```

Comment:
```suggestion
        const backTo = route?.params?.backTo;
```

We can move this to an upper scope (below `didSubscribeToReportLeavingEvents`) so we don't create this every time `onBackButtonPress` is called